Scripts/BuildPhases/CopyGutenbergJS.sh

```diff
@@ -13,6 +13,7 @@ DEST="$CONFIGURATION_BUILD_DIR/$UNLOCALIZED_RESOURCES_FOLDER_PATH"
 # Update the matching .inputs.xcfilelist when changing these
 XCFRAMEWORK_BUNDLE_ROOT="$PODS_XCFRAMEWORKS_BUILD_DIR/Gutenberg/Gutenberg.framework"
 PODS_BUNDLE_ROOT="$PODS_ROOT/Gutenberg/bundle/ios"
+LOCAL_BUNDLE="$PODS_ROOT/../../gutenberg-mobile/bundle/ios"
```

Comment:
When testing a production version of the JS bundle in local builds, I noticed we don't have a quick way to use a local bundle. For this reason, I added a third option to fetch the bundle by pointing to a local path.

Reply:
The option to fetch the bundle form `$PODS_ROOT/Gutenberg/bundle/ios` seems to be deprecated as the library is provided with the XCFramework. We could remove it.